lib/Backend/InlineeFrameInfo.cpp

```diff
@@ -172,6 +172,10 @@ void InlineeFrameRecord::PopulateParent(Func* func)
 void InlineeFrameRecord::Finalize(Func* inlinee, uint32 currentOffset)
 {
     this->PopulateParent(inlinee);
+#if TARGET_32
+    const size_t offsetMask = (~(size_t)0) >> 4;
```

Comment:
nit: Why not use uint32 here? Why hardcode 4 instead of (32 - ksizeofInlineeStartOffset) or (sizeof(uint) * CHAR_BIT - ksizeofInlineeStartOffset)?

Reply:
Yep, I should use uint32. Regarding the shift, I only want to make sure we are within the 28-bit limit on x86. `currentOffset` is uint32, so there's no point checking for 32 bit limit

---
In reply to: [204944353](https://github.com/Microsoft/ChakraCore/pull/5490#discussion_r204944353) [](ancestors = 204944353)

Reply:
Actually, regardless of which platform I want to do this check on, hardcoding 4 is probably not the best idea. I'll change that. Thanks.